src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -182,13 +182,17 @@ private static DetailAST createFakeBlockComment(String content) {
         blockCommentBegin.setType(TokenTypes.BLOCK_COMMENT_BEGIN);
         blockCommentBegin.setText("/*");
         blockCommentBegin.setLineNo(0);
-        blockCommentBegin.setColumnNo(0);
+        // -3 indentation because of BLOCK_COMMENT_BEGIN and leading astrisk
+        final int blockCommentIndent = -3;
```

Comment:
If we are changing this, we also don't need the one use variable. Just inline it.